commands/publish/index.js

```diff
@@ -237,10 +238,29 @@ class PublishCommand extends Command {
 
     return chain.then(() => {
       const count = this.packagesToPublish.length;
-      const message = this.packagesToPublish.map((pkg) => ` - ${pkg.name}@${pkg.version}`);
 
       output("Successfully published:");
-      output(message.join(os.EOL));
+
+      if (this.options.summaryFile) {
+        // create a json object and output it to a file location.
+        const filePath = this.options.summaryFile || "./output.json";
```

Comment:
`output.json` is an exceptionally generic name, and I think therefore we can do better for this default.

Maybe `lerna-publish-summary.json`?

Reply:
I think that's fair, will update when I get a chance here